requirements.txt

```diff
@@ -0,0 +1,2 @@
+Flask
```

Comment:
i was not sure if they are latest and stable 
but removing those will only install latest and stable